vector/src/main/java/im/vector/app/features/roomprofile/RoomProfileViewModel.kt

```diff
@@ -176,6 +176,36 @@ class RoomProfileViewModel @AssistedInject constructor(
             RoomProfileAction.CreateShortcut -> handleCreateShortcut()
             RoomProfileAction.RestoreEncryptionState -> restoreEncryptionState()
             is RoomProfileAction.SetEncryptToVerifiedDeviceOnly -> setEncryptToVerifiedDeviceOnly(action.enabled)
+            is RoomProfileAction.ReportRoom -> handleReportRoom(action.reason)
+        }
+    }
+
+    private fun handleReportRoom(reason: String) {
+        _viewEvents.post(RoomProfileViewEvents.Loading())
+        session.coroutineScope.launch {
+            try {
+                // When reporting a user, use the user state event if available (it should always be available)
```

Comment:
What is "the user state event"?

Reply:
The `m.room.member` event I guess?

Reply:
Yes, sorry, the comment was a copy paste error. It's gone now, with the usage of the other API.